Healthy/Classes/Modules/SavedRecipes/SavedRecipesViewController.swift

```diff
@@ -33,7 +48,42 @@ extension SavedRecipesViewController {}
 
 // MARK: - Configurations
 
-extension SavedRecipesViewController {}
+extension SavedRecipesViewController {
+    func configureNavigationBar() {
+        navigationBar.topItem?.title = "Saved recipes"
+        if let font = UIFont(name: "Poppins-Bold", size: 18) {
+            UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.font: font]
```

Comment:
This will apply this font to the whole app!

Instead, please add this to Extensions/UINavigationBar+Appearance

extension UINavigationBar {
    
    /// Apply the default navigation bar appearance
    ///
    class func applyDefaultAppearance() {
        UINavigationBar.appearance().titleTextAttributes = [.font: UIFont.mediumRegular]
    }
}

and call it in the app delegate - did finish launching with options